src/domain/UBGraphicsScene.cpp

```diff
@@ -1052,7 +1053,9 @@ void UBGraphicsScene::eraseLineTo(const QPointF &pEndPoint, const qreal &pWidth)
                 polygonItem->setNominalLine(false);
                 polygonItem->setStroke(intersectedPolygonItem->stroke());
                 polygonItem->setStrokesGroup(intersectedPolygonItem->strokesGroup());
-                intersectedPolygonItem->strokesGroup()->addToGroup(polygonItem);
+                auto stuff = intersectedPolygonItem->strokesGroup();
```

Comment:
I added this by mistake, it's some unrelated diagnosis code

Reply:
:+1: 

Thank you